http_logs/operations/default.json

```diff
@@ -89,7 +89,7 @@
           "by_hour": {
             "date_histogram": {
               "field": "@timestamp",
-              "interval": "hour"
```

Comment:
Note: this was previously interpreted as a calendar interval, but has been changed to fixed.  The old parsing logic treated "singular" values (hour, day, month, etc) as calendar if it was parseable as calendar.

So in this case it should be `calendar_interval: hour`